features/create_article_form.feature

```diff
@@ -5,6 +5,7 @@ Feature: Article creation form
 
 Background:
   Given I am on the landing page
+  And user is signed in
```

Comment:
This feature is about the show page. Restrictions to creat is out of of scope

Reply:
This is a feature file created earlier. We needed to modify it to make it pass since we added a user model.